### DISCUSSION TITLE
Builds failing for no appearant reason

### DISCUSSION THREAD

**Discussion by TaimoorHassan** [codemagic-ci-cd/codemagic-docs#2579] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

None. Because the same code may or may not work sometimes. 

### Expected results

Artifacts are built and deployed. 

### Actual results

![Screenshot 2024-06-04 at 23 08 34](https://github.com/codemagic-ci-cd/codemagic-docs/assets/8298536/62f57db6-0a4b-49e7-b167-ae4a876b7168)

Build fails with this error.

### Build id (optional)

665f52873a452dadc7b6d495

**Comment by mikhail-tokarev** [comment, marked as answer]
Hi all! We are experiencing issues with network in one of our data center. We'll post more information once we have on https://status.codemagic.io and https://github.com/orgs/codemagic-ci-cd/discussions/2568 

I'm going to close this issue to keep all communication in one place. Thank you for your understanding!

**Reply by TaimoorHassan** [reply]
Thank you.
